### ISSUE TITLE
Doesn't handle 304 responses correctly

### ISSUE THREAD

**Issue by orischwartz** [department-of-veterans-affairs/sniffybara#5]
304's return blank contents: 

```html
<html><head></head><body></body></html>
```

Can be fixed by passing options to driver:

```ruby
:phantomjs_options => ['--disk-cache=true']
```

See [this report](http://antulik.com/blog/2015/05/27/debugging-poltergeist+&cd=10&hl=en&ct=clnk&gl=us) of identical problem.
